### PR TITLE
better job handling on appveyor

### PR DESCRIPTION
**Pull request by grzanka**

### CODE REVIEW

**Comment by nbassler** [comment]
#220 is a MR, not an issue?

**Comment by grzanka** [comment]
Should be #320, my mistake

File: appveyor.yml [hunk 1]
```diff
@@ -116,8 +116,11 @@ test_script:
   - cd dist
   - python -c "import os; import pip; pip.main(['install', os.listdir('.')[0]]);"
   - cd ..
+# exclude problematic tests on 32bit platform (see https://github.com/pytrip/pytrip/issues/320)
+  - ps: $exclude_tests = If ($platform -eq "x86") {"test_converters.py"} Else {""}
 # run tests (normal py.test command results in memory error on appveyor, so we run tests file-by-file)
-  - FOR %%I in (tests\test*.py) DO python -m pytest %%I
```
Comment (grzanka) [inline appveyor.yml]: This loop was returning True even if one of the tests has failed. I had to replace it with sth more sophisticated - this time Powershell

File: tests/test_converters.py [hunk 2]
```diff
@@ -36,7 +36,7 @@
 import pytrip.utils.bevlet2oer
 import pytrip.utils.rst_plot
 
-from tests.test_base import get_files
+from tests.base import get_files
```
Comment (grzanka) [inline tests/test_converters.py]: test_base was treated as an empty tests and was failing

File: tests/test_paths.py [hunk 3]
```diff
@@ -19,6 +19,31 @@
 """
 TODO: documentation here.
 """
+import unittest
```
Comment (grzanka) [inline tests/test_paths.py]: test_paths was treated as an empty tests and was failing

**Comment by grzanka** [comment]
This PR will not solve #320, but at least any other bugs like this will be reported.

**Pull request merged**